generators/client/templates/src/main/webapp/app/layouts/navbar/navbar.html

```diff
@@ -44,39 +44,39 @@
                     </a>
                     <ul class="dropdown-menu" uib-dropdown-menu>
                         <li ui-sref-active="active" ng-switch-when="true">
-                            <a ui-sref="settings" data-toggle="collapse" data-target=".navbar-collapse.in">
+                            <a ui-sref="settings" ng-click="vm.hideNavbar()">
                                 <span class="glyphicon glyphicon-wrench"></span>&nbsp;
                                 <span translate="global.menu.account.settings">Settings</span>
                             </a>
                         </li>
                         <li ui-sref-active="active" ng-switch-when="true">
-                            <a ui-sref="password" data-toggle="collapse" data-target=".navbar-collapse.in">
+                            <a ui-sref="password" ng-click="vm.hideNavbar()">
                                 <span class="glyphicon glyphicon-lock"></span>
                                 <span translate="global.menu.account.password">Password</span>
                             </a>
                         </li>
                         <%_ if (authenticationType == 'session') { _%>
                         <li ui-sref-active="active" ng-switch-when="true">
-                            <a ui-sref="sessions" data-toggle="collapse" data-target=".navbar-collapse.in">
+                            <a ui-sref="sessions" ng-click="vm.hideNavbar()">
                                 <span class="glyphicon glyphicon-cloud"></span>&nbsp;
                                 <span translate="global.menu.account.sessions">Sessions</span>
                             </a>
                         </li>
                         <%_ } _%>
                         <li ui-sref-active="active" ng-switch-when="true">
-                            <a href="" ng-click="vm.logout()" id="logout" data-toggle="collapse" data-target=".navbar-collapse.in">
+                            <a href="" ng-click="vm.logout()" id="logout">
                                 <span class="glyphicon glyphicon-log-out"></span>&nbsp;
                                 <span translate="global.menu.account.logout">Sign out</span>
                             </a>
                         </li>
                         <li ui-sref-active="active" ng-switch-when="false">
-                            <a href="" ng-click="vm.login()" id="login" data-toggle="collapse" data-target=".navbar-collapse.in">
+                            <a href="" ng-click="vm.login();vm.hideNavbar();" id="login">
```

Comment:
You are calling hidenavbar inside the login function already so its not required again here rite?